operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -416,4 +485,272 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
                 .value(requestBaseURI + "/" + uriPath)
                 .build();
     }
+    
+    private void addIncludesSearchParameters(String compartmentType, List<String> typeParameters, MultivaluedMap<String, String> searchParameters) {
+        // Add in Location, Medication, Organization, and Practitioner resources which are pointed to
+        // from search parameters only if the request does not have a _type parameter or it does have a 
+        // _type parameter that includes these
+
+        // Remove any _includes we added last time around
+        searchParameters.remove("_include");
```

Comment:
this feels a bit odd to me.  if we're gonna clear out the existing search parameters, I think we should make that explicit in the method contract by adding some javadoc.

Reply:
alternatively we could construct a fresh searchParameters MultivaluedMap for each invocation instead of re-using that same one on each iteration (replacing just the _include value)

Reply:
I thought about this and thought a fresh MultivaluedMap might be more expensive than editing each time, but I basically just coin flipped the decision and am happy to change to your way.

Reply:
the thing that bothers me about reusing it is it feels non-obvious to me, so a source for potential future bugs.  thats why i said if you do it this way i'd want a clear javadoc stating the contract.  probably better just to construct the fresh MultivaluedMap each time (unless you have evidence of a perf issue)

Reply:
I agree with Lee on the cost of future bugs.

Reply:
OK, I have changed to construct a new MultivaluedMap each time.